tests/test_database_manager.py

```diff
@@ -42,54 +42,53 @@ class TestStuff(object):
     manager = DatabaseManager()
 
     async def test_has_table(self):
-        if await self.manager._has_table("testtablehas"):
-            await self.manager._drop_table("testtablehas")
-        assert not await self.manager._has_table("testtablehas")
-        await self.manager._create_table("testtablehas", {"string1": "VARCHAR"})
-        assert await self.manager._has_table("testtablehas")
+        if await self.manager.has_table("testtablehas"):

```

Comment:
Your tests should use a fixture rather than a class / instance attribute

Reply:
you come up with that now?

Reply:
i raised it twice before